### DISCUSSION TITLE
getting messageID

### DISCUSSION THREAD

**Discussion by JosephMasole** [python-smpplib/python-smpplib#193] [Q&A] [answered]
hello, it's my first time using this package so i wanted to know how can I get the messageID from the submit_sm resp so that I can use it to execute query_sm to obtain the delivery status. Please your immediate assistance would  be highly appreciated

**Comment by eigenein** [comment, marked as answer]
Hi @JosephMasole,

I believe this is covered in the `README`. Have you tried using the `message_id` attribute?
